saltgui/static/scripts/routes/page.js

```diff
@@ -146,9 +148,35 @@ class PageRoute extends Route {
     if(this.keysLoaded && this.jobsLoaded) this.resolvePromise();
   }
 
+  _runningJobs(data) {
+    const jobs = data.return[0];
+    for(const k in jobs)
+    {
+      const job = jobs[k];
+
+      // start with same text as for _addJob
+      let targetText = job.Target;
+      if(job["Target-type"] !== "glob" && job["Target-type"] !== "list") {
+        // note that due to bug in 2018.3, all finished jobs
+        // will be shown as if of type 'list'
+        // therefore we suppress that one
+        targetText = job["Target-type"] + " " + targetText;
```

Comment:
Wouldn't it be an idea to create a function for this? Now I see this for the third time in de codebase when I'm not mistaking

Reply:
It is in there 2 'only' times. Once for the overview; and once for the detailed view. Still reason enough to create a function for it... please wait...

Reply:
So here you should also use `window.makeTargetText` right?